### DISCUSSION TITLE
Enterprise Server requirements

### DISCUSSION THREAD

**Discussion by kristiansyrjanen** [community/community#49283] [Enterprise]
The [Doc pages ](https://docs.github.com/en/enterprise-server@3.4/admin/installation/setting-up-a-github-enterprise-server-instance/installing-github-enterprise-server-on-vmware#minimum-requirements)seem to talk about default server hardware prerequisites and then Github Actions for Enterprise Server prerequisites.
Are these differentiated, meaning that one would need e.g. core server hardware prerequisites of 8 vCPU + 48Gb and an additional 8 vCPU + 64GB for Github Actions? 
Or does it mean that the entire hardware capability needs to be at 8 vCPU + 64GB for the server to run with Github Actions enabled?

To my eyes it seems a bit confusing. Couldn't ask this through a ticket as I'm not an Enterprise customer.

Thanks in advance!

**Comment by queenofcorgis** [comment]
👋🏼 Let me talk to some folks and see what I can learn. Definitely sounds like this is a great opportunity for us to update our docs too!

**Comment by queenofcorgis** [comment]
Thank you times a million for your patience! Please refer to the hardware [prerequisites](https://docs.github.com/en/enterprise-server@3.4/admin/installation/setting-up-a-github-enterprise-server-instance/installing-github-enterprise-server-on-vmware#minimum-requirements) in GitHub Actions section to run both Actions and standard GitHub activities. If there are any follow-ups please let me know.

**Comment by kristiansyrjanen** [comment]
Hi Lili,

Thanks, but I think it's the same link that I was talking about. This is
the step that is bugging me out:

*If you plan to enable GitHub Actions for the users of your instance, more
resources are required*.

After that it lists specifications needed, but it doesn't state clearly if
it is the total needed minimum requirements along with server prerequisites
OR just additional resources on top of server installation.

Maybe it's just going a bit over my head as I'm not a native english speaker


Thanks!

\\Kristian

pe 31. maalisk. 2023 klo 22.48 Lili ***@***.***> kirjoitti:
